### PR TITLE
x11-misc/: More cleanup

### PR DESCRIPTION
**Pull request by Moiman**
@SoapGentoo

### CODE REVIEW

File: x11-misc/superswitcher/superswitcher-0.6-r1.ebuild [hunk 1]
```diff
@@ -25,16 +25,18 @@ DEPEND="${RDEPEND}
...
 		src/Makefile.am || die #338906
 
-	epatch "${FILESDIR}"/${P}-wnck-workspace.patch
-	epatch "${FILESDIR}"/${PN}-0.6-glib-single-include.patch
-	eautoreconf
-}
+	mv configure.{in,ac} || die #426262
 
-src_install() {
-	MAKEOPTS=-j1 default
```
Comment (SoapGentoo) [inline x11-misc/superswitcher/superswitcher-0.6-r1.ebuild]: you checked that the build system works in parallel?

File: x11-wm/oroborus/oroborus-2.0.20-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	x11-libs/libX11"
+DEPEND="${RDEPEND}
+	x11-proto/xproto
+	x11-proto/xextproto"
+
+DOCS=( AUTHORS ChangeLog example.${PN}rc README TODO )
+
+src_configure() {
+	econf --disable-dependency-tracking
+}
```
Comment (SoapGentoo) [inline x11-wm/oroborus/oroborus-2.0.20-r1.ebuild]: Nix `src_configure`: https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-139001r12

File: x11-misc/idesk/idesk-0.7.5-r3.ebuild [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+RDEPEND=">=media-libs/imlib2-1.4[X]
+	media-libs/freetype
+	dev-libs/libxml2
+	dev-libs/glib:2
+	x11-libs/pango
+	x11-libs/gtk+:2
+	media-libs/libart_lgpl
+	x11-libs/startup-notification"
+DEPEND="${RDEPEND}
```
Comment (a17r) [inline x11-misc/idesk/idesk-0.7.5-r3.ebuild]: It's good practise to keep dependencies sorted. ;)

File: x11-misc/superswitcher/superswitcher-0.6-r1.ebuild (hunk 1 above)
Comment (Moiman) [inline x11-misc/superswitcher/superswitcher-0.6-r1.ebuild]: Yes. I tested it.

File: x11-wm/oroborus/oroborus-2.0.20-r1.ebuild (hunk 2 above)
Comment (Moiman) [inline x11-wm/oroborus/oroborus-2.0.20-r1.ebuild]: Should I also nix src_install and gnome use flag. I doubt that its useful anymore.

Comment (SoapGentoo) [inline x11-wm/oroborus/oroborus-2.0.20-r1.ebuild]: If you like. USE flags for installing one additional file is kinda :unamused:, so I'd get rid of it too

File: x11-misc/idesk/idesk-0.7.5-r3.ebuild (hunk 3 above)
Comment (SoapGentoo) [inline x11-misc/idesk/idesk-0.7.5-r3.ebuild]: yes please, I forgot to check for this - this is part of the "unofficial" Gentoo style (first unconditional, then conditional deps, always sorted within blocks)

**Pull request closed without merging**